src/ORM/Connect/MySQLDatabase.php

```diff
@@ -65,6 +65,8 @@ class MySQLDatabase extends Database implements TransactionManager
      */
     private $transactionManager = null;
 
+    private int $transactionNesting = 0;
```

Comment:
This is to fix a dynamic property elsewhere in the file i.e. `$this->transactionNesting = ...`